velero-plugins/common/util.go

```diff
@@ -11,6 +11,12 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+// temporary keys added by mig-controller
+const (
+	transientBackupAnnotationKey = "openshift.io/migrate-type"
```

Comment:
This is the same annotation that we use as a permanent annotation on the Backup resource. We should probably use a different one just to make it clear that they are serving a different function, although from a code point of view, they won't clash since they're applied to different resource types.